Automate/RedHatConsulting_Satellite6/Integration/Satellite/DynamicDialogs.class/__methods__/get_lifecycle_environment_names_filtered_by_tags.rb

```diff
@@ -0,0 +1,173 @@
+# Populates a dynamic drop down with the Lifecycle Enviornments from a Satellite Organization
```

Comment:
@A-Beck whats the difference between this and https://github.com/RedHatOfficial/miq-RedHat-Satellite6/blob/master/Automate/RedHatConsulting_Satellite6/Integration/Satellite/DynamicDialogs.class/__methods__/get_lifecycle_environments_filtered_by_tags.rb ?

Reply:
This is different because it will populate a dropdown with the lifecycle environment name instead of the the lifecycle environment ID.

I've update the comments at the beginning of the file to reflect this.

Reply:
woudn't these methods be identical in every other way then ID vs name? Some way to use just one method and have alike a method attribute that lets you choose whether name or ID? maybe im just being over kill.

Reply:
@itewk yeah, that is the better approach. I've updated the other method to take a input that defaults to 'id' to avoid duplicated code.

I also updated the other dialog element to be able to take an env name or an env id to filter providers on. Could you review again, once you get a moment?